src/imgs/src/vram_map.svg

```diff
@@ -0,0 +1,230 @@
+		<text x="340" y="154" class="centered descr" id="tile_id"></text>
+		<text x="340" y="174" class="centered descr">Addr:</text>
+		<text x="340" y="194" class="centered descr" id="tile_addr"></text>
+	</g>
+	<g class="hover">
+		<use x="100" y="308" href="#tilemap" class="hover" onmouseenter="tmap(0x9800, 0)"/>
+		<text x="90" y="404" class="right descr">Tile map 0</text>
+		<use x="100" y="500" href="#tilemap" class="hover" onmouseenter="tmap(0x9C00, 0)"/>
+		<text x="90" y="596" class="right descr">Tile map 1</text>
+		<use x="388" y="308" href="#tilemap" class="hover" onmouseenter="tmap(0x9800, 1)"/>
+		<text x="590" y="404" class="descr">Attr map 0</text>
+		<use x="388" y="500" href="#tilemap" class="hover" onmouseenter="tmap(0x9C00, 1)"/>
+		<text x="590" y="596" class="descr">Attr map 1</text>
+	</g>
+	<g>
+		<text x="340" y="470" class="centered descr" id="map_what"></text>
+		<text x="338" y="490" class="right descr">X =</text><text x="342" y="490" class="descr" id="map_x"></text>
+		<text x="338" y="510" class="right descr">Y =</text><text x="342" y="510" class="descr" id="map_y"></text>
+		<text x="340" y="530" class="centered descr" id="map_addr"></text>
+	</g>
+	<script>// <![CDATA[
+"use strict";
+/*
+ * Use of `on<event>` properties is normally discouraged, however they *have to* be used here.
+ *
+ * For the sake of maintainability and the SVG's file size, the tilemaps are made of nested `<use>`
+ * elements. However, those elements have a "closed root", thus the "inside" elements are *not*
+ * exposed to anything outside of them.
+ *
+ * I have tried putting `<script>` blocks inside of the "template" element, but they appear to only
+ * run on the template, not within any of its clones.
+ * The only remaining thing is the `on*` attributes, which *do* appear to be cloned!
+ */
+const toHex = (num, nbDigits) => num.toString(16).toUpperCase().padStart(nbDigits, '0');
+const svg = document.getElementById("vram_map");
+/*
+ * Tile blocks
+ */
+// Since the hover events are dispatched to several separate handlers, we need something to
+// centralize all of their information.
+let tileInfo = { x: null, y: null, baseAddr: null };
+const updateTileID = () => {
+	// Do nothing if all of the handlers haven't fired at least once yet.
+	// Check all properties to avoid relying on the order in which the events are triggered.
+	if (Object.values(mapInfo).includes(null)) {
+		return;
+	}
+	const tileID = tileInfo.y * 16 + tileInfo.x + (tileInfo.baseAddr >> 4 & 0x80);
+	svg.getElementById("tile_id").textContent = `$${toHex(tileID, 2)}`;
+	const tileAddr = tileInfo.baseAddr + (tileInfo.y * 16 + tileInfo.x) * 16;
+	svg.getElementById("tile_addr").textContent = `${toHex(tileAddr, 4)}–${toHex(tileAddr + 15, 4)}`;
+};
+// These are event handlers, called into by `onmouseenter` attributes within `vram_map.svg`.
+// Have each hover event attempt to recalc the tile ID; this will cause up to 2 unnecessary recalcs,
```

Comment:
English language convention pedantry.
```suggestion
// Have each hover event attempt to recalc the tile ID; this will cause up to two unnecessary recalcs,
```